test/classes/ferguson/delete-free/owned-shared-fields2.chpl

```diff
@@ -1,23 +1,30 @@
 use OwnedObject;
 use SharedObject;
 
+pragma "use default init"
```

Comment:
I haven't gone looking to figure out what this pragma is for, but can you explain why these classes and records need this pragma?

Reply:
The pragma enables the default initializer (vs the old default constructor). The old default constructor is not working for owned fields, but we're trying to move away from it presently, so that didn't seem worth digging in to. So I just marked it as "use default init" to require the new default initializer code is used.